test-js/test-glyph-controller.js

```diff
@@ -67,6 +73,17 @@ describe("glyph-controller Tests", () => {
     expect(staticGlyphController.xAdvance).to.equal(170);
   });
 
+  it("get empty StaticGlyphController xAdvance", () => {
+    const sgObj = makeTestEmptyStaticGlyphObject();
+    const staticGlyph = StaticGlyph.fromObject(sgObj);
+    const staticGlyphController = new StaticGlyphController(
+      "dummy",
+      staticGlyph,
+      undefined
+    );
+    expect(staticGlyphController.xAdvance).to.equal(170);
```

Comment:
Please add the test lines from the leftMargin and rightMargin tests here. They don't all need their own test func.

Reply:
Done.